dagger-core/src/main/java/io/odpf/dagger/core/processors/external/pg/PgSourceConfigBuilder.java

```diff
@@ -0,0 +1,106 @@
+package io.odpf.dagger.core.processors.external.pg;
+
+import java.util.Map;
+
+public class PgSourceConfigBuilder {
```

Comment:
If this builder is used for the only tests, can we move it to the test package?

Reply:
since the focus of the pr is to refactor tests so restricted the usage of this class for test code only for now. Will start using this builder class in main code in future